server/webapp/WEB-INF/rails.new/app/views/preferences/notifications.html.erb

```diff
@@ -0,0 +1,27 @@
+<header class="page-header no-buttons">
+  <h1>Preferences</h1>
+</header>
+
+<ul class="sub-tab-navigation">
+  <li class="current"><a>Notifications</a></li>
+  <li><%= link_to "OAuth Access Token", oauth_engine.user_tokens_index_path %></li>
+</ul>
+<div class="sub-tab-content">
+
+  <% unless go_config_service.is_smtp_enabled %>
+    <div class="warning">
+      <i class="fa fa-exclamation-triangle"></i>
+      SMTP settings are currently not configured. If you are the administrator, you can configure email support at <%= link_to "Server Configuration", config_view_path %>.
+    </div>
+  <% end %>
+
+  <div id="validations"></div>
+
+  <h2 class="my-notifications">My Notifications</h2>
+
+  <div id="notification-prefs" class="notification-content"
+       data-user-url="<%= apiv4_current_user_path(current_user.username.to_s) %>"
+       data-filters-url="<%= apiv4_notification_filters_path(current_user.username.to_s) %>"
+       data-pipelines="<%= @pipelines.to_json %>">
```

Comment:
Yeah, it might as the implementation looks similar. I'm a bit on the fence in regards to switching to an AJAX request to populate data that won't be altered by the page itself (and likely won't change often), though I can see the argument from a consistency viewpoint. I suppose an extra request on the page won't be that bad.